buildshiprun/handler.go

```diff
@@ -33,6 +33,14 @@ func Handle(req []byte) string {
 		log.Panic(eventErr)
 	}
 
+	defer func() {
```

Comment:
This is an interesting approach - I need to look into panic recovery in Go and comment back on this.